projects/igniteui-angular/src/lib/grids/public_api.ts

```diff
@@ -13,7 +13,7 @@ export * from './grid-base.directive';
 export * from './grid.common';
 export * from './grid-public-row';
 export * from './grid-public-cell';
-export { CellType, RowType, IGX_GRID_BASE, ValidityStatus, IRecordValidationState, IFieldValidationState } from './common/grid.interface';
+export { CellType, RowType, IGX_GRID_BASE, ValidationStatus, IGridValidationState, IRecordValidationState, IFieldValidationState } from './common/grid.interface';
```

Comment:
One of the event arg type: `IGridValidationStatusEventArgs` is missing here. It should also be exported.

Reply:
Actually, `IGridFormGroupCreatedEventArgs` is missing as well, good catch :)